src/transactions/feed/queryHelper.ts

```diff
@@ -344,16 +361,16 @@ async function queryChainTransactionsFeed({
       tokenTransactionsV3: {
         ...body.data.tokenTransactionsV3,
         transactions: body.data.tokenTransactionsV3.transactions.map((tx) => {
-          return { ...tx, network }
+          return { ...tx, networkId }
         }),
       },
     },
   }
 }
 
 export const TRANSACTIONS_QUERY = `
-  query UserTransactions($address: Address!, $localCurrencyCode: String, $afterCursor: String, $network: Network!) {
-    tokenTransactionsV3(address: $address, localCurrencyCode: $localCurrencyCode, afterCursor: $afterCursor, network: $network) {
+  query UserTransactions($address: Address!, $localCurrencyCode: String, $afterCursor: String, $networkId: NetworkId!) {
```

Comment:
N.B to @cajubelt -- I'm assuming that the parameter for the blockchain-api endpoint is going to be called `networkId` with an enum name of `NetworkId` in the GraphQL schema.